docs/build-playbook-cr.md

```diff
@@ -0,0 +1,33 @@
+| [Home](../README.md) |
+|----------------------|
+
+## CR for Building a new Playbook
+
+This task entails following sub-tasks:
+
+1. Application administrator creates a new CR for building a new playbook. For creating a new CR refer to the [New Change Request from Production Environment](#new-change-request-from-production-environment).
+
+2. Content Developer creates a new playbook. For creating new playbooks, refer to [Introduction to playbooks](https://docs.fortinet.com/document/fortisoar/7.3.1/playbooks-guide/331279/introduction-to-playbooks). 
+
+> [!NOTE]
+> Following actions must be carried out if the playbook requires a new playbook collection:
```

Comment:
Following actions must be carried out if content developer wants to pushed the a new playbook collection:

Reply:
Fixed